src/MoBi.Presentation/Presenter/EditFavoritesPresenter.cs

```diff
@@ -197,5 +141,22 @@ public void Handle(RemovedEvent eventToHandle)
 
          refresh();
       }
+
+      public void MoveUp() => moveFavorites(x => x.MoveUp);
+
+      public void MoveDown() => moveFavorites(x => x.MoveDown);
+
+      private void moveFavorites(Func<IFavoriteTask, Action<IEnumerable<string>>> moveActionFunc)
+      {
+         var selectedParameters = _editParameterListPresenter.SelectedParameters;
```

Comment:
Same code as in PKSim. Not sure how to not duplicate the code since we have complete different presenters